js/utils/constants.js

```diff
@@ -918,6 +918,115 @@
        */
       VC_TAG_FOUND: 'videoTagFound',
 
+      /**
+       * This event is triggered when an ad sdk has been loaded successfully. the handler is called with:
+       * <ul>
+       *   <li>The ad plugin loaded.</li>
+       * </ul>
+       * @event OO.EVENTS#AD_SDK_LOADED
+       */
+      AD_SDK_LOADED: 'adSdkLoaded',
+
+      /**
+       * This event is triggered when there is an failure to load the ad sdk.        
```

Comment:
For consistency, can you add in 'The handler is called with the following arguments:' to all the new messages below.

Reply:
Fixed